src/test/ui/feature-gates/bench.stderr

```diff
@@ -0,0 +1,10 @@
+error: use of unstable library feature 'test': `bench` is a part of custom test frameworks which are unstable
+  --> $DIR/bench.rs:1:3
+   |
+LL | #[bench]
+   |   ^^^^^
+   |
+   = note: `#[deny(soft_unstable)]` on by default
```

Comment:
Can you record this as a C-future-compatibility lint in the compiler and file an associated issue (with the labels, etc.) to track this?